policy/datasets.yaml

```diff
@@ -113,6 +113,14 @@ deepCoaddId:
     storage: ignored
     python: lsst.daf.base.PropertySet
     template: ignored
+deepCoadd_band:
+    description: >
+      The generic band associated with "filter" when needed but using Gen2.
+      TEMPORARY: to be removed along with Gen2.
```

Comment:
This is a bit redundant, as this file (like all mapper policy files) will be removed in Gen2.